charts/testkube-api/templates/servicemonitor.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   {{- if .Values.prometheus.interval }}
   - interval: {{ .Values.prometheus.interval }}
   {{- end }}
-    port: http
+    targetPort: 8088
```

Comment:
looks good, should it be taken from Values. @ypoplavs ?

Reply:
@gberche-orange `targetPort` can be referenced as "{{ .Values.service.port }}"

Reply:
tahnks @ypoplavs , fixed in c309958457b7bfbdc50a5ad85ec598c182267ee2